### PR TITLE
Master pyopenms

### PR DESCRIPTION
**Pull request by hroest**
merge changes from 2.3 release (including tooling for release)

### CODE REVIEW

File: src/pyOpenMS/dist-scripts/create-manylinux.sh [hunk 1]
```diff
@@ -4,12 +4,26 @@
...
+#   sudo docker run --net=host -v `pwd`:/data hroest/manylinux_qt_contrib:v2 /bin/bash /data/create-manylinux.sh
 #
 
-git clone -b Release2.2.0 https://github.com/OpenMS/OpenMS.git
+set -e
+
+# Note that we cannot use git any more as it refuses to communicate with
+# github, but we have a patched version of wget with a newer OpenSSL version
+# capable of downloading the release tar file.
+wget https://github.com/OpenMS/OpenMS/releases/download/Release2.3.0/OpenMS-2.3.0-src.tar.gz -O OpenMS-2.3.0-src.tar.gz
```
Comment (timosachsenberg) [inline src/pyOpenMS/dist-scripts/create-manylinux.sh]: can you add to https://github.com/OpenMS/OpenMS/wiki/Preparation-of-a-new-OpenMS-release that this file needs to be adapted

File: src/pyOpenMS/doc/Manual.tex [hunk 2]
```diff
@@ -24,7 +24,7 @@
 \usepackage{tocloft}% http://ctan.org/pkg/tocloft
 \setlength{\cftsubsecnumwidth}{3.5em}% Set length of number width in ToC for \subsection
 
-\title{pyOpenMS }
+\title{pyOpenMS 2.3}
```
Comment (timosachsenberg) [inline src/pyOpenMS/doc/Manual.tex]: also add to https://github.com/OpenMS/OpenMS/wiki/Preparation-of-a-new-OpenMS-release

**Comment by jpfeuffer** [comment]
Or can you configure it via CMake?

File: src/pyOpenMS/dist-scripts/create-manylinux.sh (hunk 1 above)
Comment (hroest) [reply src/pyOpenMS/dist-scripts/create-manylinux.sh]: done

@grosenberger can you comment on the OSX section here: https://github.com/OpenMS/OpenMS/wiki/Preparation-of-a-new-OpenMS-release#pyopenms_release

**Comment by hroest** [comment]
> Or can you configure it via CMake?

not at the moment as its not under CMake and is build manually, but eventually yes, that is the goal. For now, lets do it manually.

**Comment by jpfeuffer** [comment]
Yes, I realized that too. Makes sense.

**Comment by hroest** [comment]
since these are only scripts used for the release, I think we can merge here.

**Pull request merged**